instrumentation/storm/javaagent/src/main/java/io/lumigo/javaagent/instrumentation/storm/StormExecutorTransferInstrumentation.java

```diff
@@ -0,0 +1,82 @@
+ */
+package io.lumigo.javaagent.instrumentation.storm;
+
+import static io.lumigo.javaagent.instrumentation.storm.StormSingleton.stormExecutorInstrumenter;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.javaagent.bootstrap.Java8BytecodeBridge;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeTransformer;
+import io.opentelemetry.semconv.SemanticAttributes;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+import org.apache.storm.tuple.AddressedTuple;
+
+public class StormExecutorTransferInstrumentation implements TypeInstrumentation {
+
+  @Override
+  public ElementMatcher<TypeDescription> typeMatcher() {
+    return named("org.apache.storm.executor.ExecutorTransfer");
+  }
+
+  @Override
+  public void transform(TypeTransformer typeTransformer) {
+    typeTransformer.applyAdviceToMethod(
+        named("tryTransfer"),
+        io.lumigo.javaagent.instrumentation.storm.StormExecutorTransferInstrumentation.class
+                .getName()
+            + "$StormTryTransferAdvice");
+  }
+
+  @SuppressWarnings("unused")
+  public static class StormTryTransferAdvice {
+    @Advice.OnMethodEnter(suppress = Throwable.class)
+    public static void onEnter(@Advice.Argument(0) AddressedTuple addressedTuple) {
+      Context parentContext = Java8BytecodeBridge.currentContext();
+
+      if (!stormExecutorInstrumenter().shouldStart(parentContext, addressedTuple)
+          || (addressedTuple.tuple.getMessageId().toString().contains("{}"))) {
+        // We don't want to start a new span if the message id is empty
+        return;
+      }
+
+      // Make sure the parent span is active on the Context when you start the new span
+      try (Scope scope = parentContext.makeCurrent()) {
+        Context context = stormExecutorInstrumenter().start(parentContext, addressedTuple);
+        final Span span = Java8BytecodeBridge.spanFromContext(context);
+        span.setAttribute("service.name", StormUtils.getServiceName());
+        span.setAttribute(SemanticAttributes.THREAD_NAME, StormUtils.getThreadName());
+        span.setAttribute(
+            SemanticAttributes.MESSAGING_MESSAGE_ID, StormUtils.getMessageId(addressedTuple.tuple));
+        span.setAttribute(
+            AttributeKey.stringArrayKey("storm.tuple.values"),
+            StormUtils.getValues(addressedTuple.tuple));
+        span.setAttribute(
+            SemanticAttributes.MESSAGING_DESTINATION_NAME,
+            StormUtils.getDestComponent(addressedTuple));
+        stormExecutorInstrumenter().end(context, addressedTuple, null, null);
+      }
+    }
+  }
+}
```

Comment:
is there any response value for a bolt? 
(specifically referring to the last bolts of the sequence)